backend/editor/models.py

```diff
@@ -16,3 +19,68 @@ class Header(Marginal):
 
 class Footer(Marginal):
     pass
+
+
+# Models for FastAPI
+
+class CommonParameters:
+    branch = "branch"
+    taxonomy_name = "taxonomy_name"
+
+class Entry:
+    default: []

```

Comment:
Here is you problem that makes it raise. you put [] as if it was a type ! (typo)
```suggestion
    default = []
```

But maybe you should give the type (and what is inside the list).